fstest/fstests/fstests.go

```diff
@@ -163,7 +163,13 @@ func PutTestContentsMetadata(ctx context.Context, t *testing.T, f fs.Fs, file *f
 		in := io.TeeReader(buf, uploadHash)
 
 		file.Size = int64(buf.Len())
-		obji := object.NewStaticObjectInfo(file.Path, file.ModTime, file.Size, true, nil, nil)
+		// Callers don't always set up file.Hashes, so it is an empty map, but NewStaticObjectInfo will only create
```

Comment:
This stops us being able to pass an empty map for no hashes are supported.

Can't you adjust the caller instead?

Reply:
Yep - my goal here was to minimize the code change, but adjusting the calling code to do the right thing would be better.